### PR TITLE
Handicap new texts

### PR DESCRIPTION
**Pull request by edellucien**
# How to

Ticket: TRAFHAND-102

New handicap texts

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] The title means something for a human being.
- [ ] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js [hunk 1]
```diff
@@ -44,7 +46,7 @@ const descriptions = {
   },
   fr: {
     barrierfree:
-      ' dans lesquelles il est possible de se déplacer en toute autonomie.',
+      'Gares entièrement ou majoritairement accessibles, où il est possible de se déplacer de manière autonome',
```
Comment (gevos) [inline src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js:49]:
```suggestion
      'Gares entièrement ou majoritairement accessibles, où il est possible de se déplacer de manière autonome.',
```

File: src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js [hunk 2]
```diff
@@ -14,14 +14,15 @@ const propTypes = {
...
+  barrierfree: '',
   notBarrierfree: 'ch.sbb.nichtbarrierfreierbahnhoefe',
   stuetzpunkt: 'bahnhof_plural',
 };
 
 const descriptions = {
   de: {
-    barrierfree: ' an denen autonomes Reisen möglich ist.',
+    barrierfree:
+      'Ganz oder hauptsächlich barrierefreie Bahnhöfen an denen das autonome Reisen möglich ist.',
```
Comment (gevos) [inline src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js:25]:
```suggestion
      'Ganz oder hauptsächlich barrierefreie Bahnhöfe, an denen das autonome Reisen möglich ist.',
```

Comment (gevos) [inline src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js:25]: Replace
`Nicht barrierefreie Bahnhöfe an denen automes Reisen nicht möglich ist.`
by
`Nicht barrierefreie Bahnhöfe, an denen autonomes Reisen nicht möglich ist.`

Add comma as well as `autonomes`.

File: src/popups/HandicapPopup/HandicapPopup.js [hunk 3]
```diff
@@ -61,10 +61,13 @@ function HandicapPopup({ feature }) {
           ? `${t('behig_nach')} 2026`
           : properties.prognose;
       return `${t(
-        'Dieser Bahnhof wird barrierefrei umgebaut (Prognose)',
-      )}: ${prognose}.`;
+        `Dieser Bahnhof wird vollständig angepasst`,
+      )}: ${prognose} (${t('Prognose')}).`;
     }
-    return t('Dieser Bahnhof wird nicht baulich angepasst.');
+    if (status === 'NICHT OK') {
```
Comment (gevos) [inline src/popups/HandicapPopup/HandicapPopup.js:67]: Is it really `NICHT OK`? In the behig table in the database `NICHT OK` does not exist but `BLEIBEN NICHT OK`.

**Review by gevos — CHANGES_REQUESTED** [review]
Some remarks and questions.

File: src/popups/HandicapPopup/HandicapPopup.js (hunk 3 above)
Comment (edellucien) [reply src/popups/HandicapPopup/HandicapPopup.js:67]: That's right! Good catch :)

File: src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js (hunk 2 above)
Comment (edellucien) [reply src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js:25]: But we do not write:
'Nicht barrierefreie Bahnhöfe an denen automes Reisen nicht möglich ist.'
but 
'Ganz oder hauptsächlich barrierefreie Bahnhöfen an denen das autonome Reisen möglich ist.'

File: src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js [hunk 4]
```diff
@@ -14,14 +14,15 @@ const propTypes = {
...
     notBarrierfree: ' an denen automes Reisen nicht möglich ist.',
```
Comment (gevos) [inline src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js:26]:
```suggestion
    notBarrierfree: ', an denen autonomes Reisen nicht möglich ist.',
```

File: src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js (hunk 2 above)
Comment (gevos) [reply src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js]: but here https://github.com/geops/trafimage-maps/blob/a914108897c6052561a09593b2ff3cbd3f2ecb61/src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js#L26, see suggestion below

**Review by gevos — CHANGES_REQUESTED** [review]

**Comment by edellucien** [comment]
Thanks for the comments. Code updated.

**Review by gevos — APPROVED** [review]
LGTM!

**Pull request merged**